src/router.js

```diff
@@ -33,7 +33,7 @@ Vue.use(Router)
 
 const router = new Router({
 	mode: 'history',
-	base: linkTo('calendar', 'index.php'),
+	base: generateUrl('/apps/calendar', ''),
```

Comment:
The second parameter should be an object or undefined, so you can omit it here: https://github.com/nextcloud/nextcloud-router/blob/master/lib/index.ts#L54